libvast/vast/logger.hpp

```diff
@@ -75,12 +62,83 @@
             [func_name_] { VAST_DEBUG("EXIT {}", func_name_); });              \
         }(__VA_ARGS__);
 
+#  define VAST_TRACE(...)                                                      \
+    SPDLOG_LOGGER_TRACE(::vast::detail::logger(), __VA_ARGS__)
+
 #else // VAST_LOG_LEVEL > VAST_LOG_LEVEL_TRACE
 
 #  define VAST_TRACE_SCOPE(...) VAST_DISCARD_ARGS(__VA_ARGS__)
 
+#  define VAST_TRACE(...) VAST_DISCARD_ARGS(__VA_ARGS__)
+
 #endif // VAST_LOG_LEVEL > VAST_LOG_LEVEL_TRACE
 
+#if VAST_LOG_LEVEL >= VAST_LOG_LEVEL_DEBUG
+
+#  define VAST_DEBUG(...)                                                      \
+    SPDLOG_LOGGER_TRACE(::vast::detail::logger(), __VA_ARGS__)
+
+#else // VAST_LOG_LEVEL > VAST_LOG_LEVEL_DEBUG
```

Comment:
```suggestion
#else // VAST_LOG_LEVEL < VAST_LOG_LEVEL_DEBUG
```
Same for the other instances of this comment.

Reply:
Done